Source/PluginProcessor.cpp

```diff
@@ -251,32 +255,9 @@ void PinkTromboneAudioProcessor::processBlock (AudioBuffer<float>& buffer, MidiB
 	}
 }
 
-void PinkTromboneAudioProcessor::timerCallback()
-{
-	if (this->attackCounter < this->attackLength)
-	{
-		this->constrictionY -= this->attackStep;
-		this->attackCounter += 10;
-	} else {
-		if (this->decayCounter < this->decayLength)
-		{
-			this->constrictionY += this->decayStep;
-			this->decayCounter += 10;
-		}
-		else {
-			stopTimer();
-		}
-	}
-}
-
-void PinkTromboneAudioProcessor::envelope()
+void PinkTromboneAudioProcessor::envelope(float sampleVal)

```

Comment:
I don't love that this function is just called `envelope`. Not only is it super vague, but also it doesn't really describe that the function does. Something like "interpolateConstrictions" or "constrictionApplyEnvelope" would be more descriptive.